### DISCUSSION TITLE
How can i use tailwindcss v3 by using @tailwindcss/postcss7-compat

### DISCUSSION THREAD

**Discussion by yogesh7401** [tailwindlabs/tailwindcss#9432] [Help] [answered]
Below is the tailwind version I'm using in one of my project created using create-react-app
```json
{
    "@tailwindcss/postcss7-compat": "^2.2.17",
    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
}
```
<code>How can i use tailwindcss v3 in my app</code>

**Comment by AviAvinav** [comment, marked as answer]
To use tailwindcss v3, you will have to upgrade to the new version, you are currently on a version that is a part of tailwindcss v2 for postcss7 (in case, you ran into an error with postcss8).

To upgrade just run:

`npm i -D tailwindcss postcss autoprefixer`

and remove `@tailwindcss/postcss7-compat`

I have used `npm` but you can use any package manager you are using.

Once, this is done follow the [migration guide](https://tailwindcss.com/docs/upgrade-guide#migrating-to-the-jit-engine) to change your `tailwind.config.js`.

I can help you out with that if you can share your tailwind config.

**Reply by yogesh7401** [reply]
Thanks @AviAvinav
This what i did and the issue is in Storybook

**Reply by AviAvinav** [reply]
is it resolved now?

**Reply by yogesh7401** [reply]
Yes it resolved ✨
